src/armature/Generator.ts

```diff
@@ -232,9 +259,24 @@ export class GeneratorInstance {
  * definition for the same rule.
  */
 export class Generator {
+    public readonly wrapUpRules: { [name: string]: GeneratorFn } = {};
+    public readonly notSkeletonComponentNames: Set<string> = new Set<string>();
```

Comment:
nit: can you call this `postSkeletonComponentNames` for consistency with `postSkeletonSpawnPoints`?